src/components/Event.js

```diff
@@ -0,0 +1,14 @@
+function subscribe(eventName, listener) {
+  document.addEventListener(eventName, listener);
+}
+
+function unsubscribe(eventName, listener) {
+  document.removeEventListener(eventName, listener);
+}
+
+function publish(eventName, data) {
```

Comment:
convert from `function` to `export const <functionName> = `

Reply:
Done